### ISSUE TITLE
Syntax coloring: screen parameters are not colored

### ISSUE THREAD

**Issue by hsandt** [koroshiya/Sublime-Renpy#12]
**Repro**

Enter:

```rpy
screen gear_button(_text="NULL", _size=0.4, _xpos=0, _ypos=0, _action = NullAction(), _default_focus = None):
```

**Actual**: all parameters and default values are shown in standard color (here, white)

![image](https://github.com/koroshiya/Sublime-Renpy/assets/1067772/19b0847d-0562-44ac-931a-df9c4888b25a)

**Expected**: color parameters and default values for better readability.

For instance, look at how GitHub itself colorizes above (not perfect as `screen` and parameter names are still white, but better), or Python code below:

```py
def init(_text="NULL", _size=0.4, _xpos=0, _ypos=0, _action = NullAction(), _default_focus = None):
```

![image](https://github.com/koroshiya/Sublime-Renpy/assets/1067772/b123db24-311f-46a6-bed0-50700e2b1ca2)

**Comment by koroshiya** [comment]
I think I've got it. This should be done in commit 496d128a0a8a2b9e87e45f5ab427199d90de52cf
I'll do some more testing tomorrow before releasing anything.

**Comment by koroshiya** [comment]
This is now released in v1.2.18
